cogs/games.py

```diff
@@ -23,15 +23,174 @@
+                    timeout_dict[curr_msg.author] = time()
+                    game_state.add_msg(
+                        f"{curr_msg.author}, '{curr_guess}' was already guessed"
+                    )
+                    await ctx.send(embed=game_state.embed)
+                elif curr_guess in game_state.not_guessed:
+                    continue_game = game_state.correct(curr_guess)
+                    game_state.add_msg(
+                        f"{curr_msg.author} guessed '{curr_guess}' correctly!")
+                    await ctx.send(embed=game_state.embed)
+                    if not continue_game:
+                        winner = curr_msg.author
+                        if command is None:
+                            game_state.field_name = f"hangman (category: {pretty_name})"
+                        game_state.add_msg(
+                            f"{winner} finished solving the hangman!")
+                        if game_state.img:
+                            game_state.embed.set_image(url=game_state.img)
+                        await ctx.send(embed=game_state.embed)
+                        await ctx.send(
+                            f"congratulations `{winner}`, you solved the hangman, "
+                            f"earning you {self.hm_norm_win} cheeps")
+                        break
+                else:
+                    timeout_dict[curr_msg.author] = time()
+                    continue_game = game_state.mistake(curr_guess)
+                    game_state.add_msg(
+                        f"{curr_msg.author} guessed '{curr_guess}' wrong!")
+                    await ctx.send(embed=game_state.embed)
+                    if not continue_game:
+                        if command is None:
+                            game_state.field_name = f"hangman (category: {pretty_name})"
+                        game_state.add_msg(
+                            f"{curr_msg.author} used your last chance!")
+                        if game_state.img:
+                            game_state.embed.set_image(url=game_state.img)
+                        await ctx.send(embed=game_state.embed)
+                        await ctx.send(
+                            f"sorry everyone, `{curr_msg.author}` used your "
+                            f"last chance, the right answer was `{game_state.word}`"
+                        )
+                        break
+            else:
+                game_state.add_msg(
+                    f"{curr_msg.author} you cannot guess right now!")
+                await ctx.send(embed=game_state.embed)
+        if winner is not None:
+            conn = sqlite3.connect(self.bot.config.db_path)
+            await self.bot.get_cog("Currency").create_bank_transaction(
+                conn.cursor(), winner,
+                self.hm_cool_win if cool_win else self.hm_norm_win,
+                HANGMAN_REWARD, {"cool": cool_win})
+            conn.commit()
```

Comment:
missing `conn.close()` (still would implicitly close, but better to be explicit here)